core/src/Kokkos_OpenMPTargetSpace.hpp

```diff
@@ -51,13 +51,12 @@
 #include <typeinfo>
 
 #include <Kokkos_Core_fwd.hpp>
-
-#ifdef KOKKOS_ENABLE_OPENMPTARGET
-
 #include <OpenMPTarget/Kokkos_OpenMPTarget_Error.hpp>
 #include <Kokkos_HostSpace.hpp>
 #include <omp.h>
 
+#ifdef KOKKOS_ENABLE_OPENMPTARGET
```

Comment:
Why did you change?  Don't we need to protect the include of `<omp.h>`?

Reply:
Right, dint notice that, <omp.h> came out of the `ifdef` . Fixed it.